packages/KTable/KTable.vue

```diff
@@ -662,6 +664,10 @@ export default defineComponent({
       }
     }
 
+    const getTestIdString = (message) => {
+      return message.toLowerCase().replaceAll(' ', '-')
```

Comment:
suggestion: Just in case someone passes in something crazy. This will replace any non-alphanumeric character.
```suggestion
      return message.toLowerCase().replace(/[^[a-z0-9]/gi, '-')
```

Reply:
@adamdehaven good suggestion, I committed it via GitHub, so let's see if things are happy now